project/Dependencies.scala

```diff
@@ -4,6 +4,7 @@ import sbt.librarymanagement.syntax.ExclusionRule
 
 object Dependencies {
   val attoCore = "org.tpolecat" %% "atto-core" % "0.8.0"
+  val bcprovJdk15to18 = "org.bouncycastle" % "bcprov-jdk15to18" % "1.67"
```

Comment:
FYI, maybe we can remove `bouncycastle` dependency if covert key

https://github-api.kohsuke.org/githubappjwtauth.html

> ### Converting the private key into a Java friendly format
>
> Note: GitHub let's you download the GitHub App private key in the PEM format which isn't natively supported by the JVM unless you leverage a third-party library such as BouncyCastle. In this guide we will convert it to DER using the openssl utility.
> 
> `openssl pkcs8 -topk8 -inform PEM -outform DER -in ~/github-api-app.private-key.pem -out ~/github-api-app.private-key.der -nocrypt`

Reply:
I don't mind the extra dependency since it is not a lot of code for the PEM to DER conversion.